### PR TITLE
Fix git_dir to handle empty path

### PR DESCRIPTION
**Pull request by mckellyln**
@junegunn here is the PR after help from others.

### CODE REVIEW

File: plugin/gv.vim [hunk 1]
```diff
@@ -211,11 +211,8 @@ function! s:setup(git_dir, git_origin)
   let b:git_dir = a:git_dir
 endfunction
 
-function! s:git_dir()
-  if empty(get(b:, 'git_dir', ''))
-    return fugitive#extract_git_dir(expand('%:p'))
-  endif
-  return b:git_dir
+function! s:git_dir() abort
+  return FugitiveGitDir()
```
Comment (junegunn) [inline plugin/gv.vim]: Thanks. We can just replace `s:git_dir()` calls with `FugitiveGitDir()` and remove the function.

Comment (mckellyln) [reply plugin/gv.vim]: You mean like this ?
```
  let git_dir = FugitiveGitDir()
```
ok, sure, hang on.

Comment (mckellyln) [reply plugin/gv.vim]: or do you mean remove it as an arg to s:setup() and everywhere else also ?

**Comment by junegunn** [comment]
Merged, thanks!

**Pull request merged**